src/vcf/report_writer.cpp

```diff
@@ -0,0 +1,120 @@
+//
+// Created by jmmut on 25/05/16.
+//
+
+#include "vcf/report_writer.hpp"
+
+namespace ebi
+{
+  namespace vcf
+  {
+    SqliteReportWriter::SqliteReportWriter(std::string db_name) : db_name{db_name}, current_transaction_size{0},
+                                                      transaction_size{1000000}, sleep_time{500}
+    {
+        int rc = sqlite3_open(db_name.c_str(), &db);
```

Comment:
It should be checked whether the database already exists (probably from a previous run).